### ISSUE TITLE
nonexistent #includes: Unix2Nt.hpp, startsock.hpp, iRODSNtUtil.hpp

### ISSUE THREAD

**Issue by SwooshyCueb** [irods/irods#6234]
- [x] main
- [x] 4-3-stable

---

In a few locations, we `#include` a few headers (`Unix2Nt.hpp`, `iRODSNtUtil.hpp`, and `startsock.hpp`) that have not existed since iRODS 3:
https://github.com/irods/irods/blob/db876cc051697b81c2f05e8bf62274f1c067a82c/lib/core/src/obf.cpp#L57-L58
https://github.com/irods/irods/blob/db876cc051697b81c2f05e8bf62274f1c067a82c/lib/core/src/rcConnect.cpp#L6
https://github.com/irods/irods/blob/db876cc051697b81c2f05e8bf62274f1c067a82c/lib/core/src/rcMisc.cpp#L41
https://github.com/irods/irods/blob/db876cc051697b81c2f05e8bf62274f1c067a82c/server/api/src/rsExecCmd.cpp#L10

These `#include` lines are inside precompiler conditional blocks that are not enabled during normal builds, so they do not currently cause compilation failures. It looks like they are left over from when iRODS ran on Windows.

**Comment by trel** [comment]
Yes, please to remove.    We have no intention of running the server on Windows in the future.
